PrivateHeaders/APPX/ZIP.h

```diff
@@ -182,6 +184,44 @@ namespace appx {
         }
     };
 
+    inline
+    bool
+    _IsAPPXFile(const std::string &inputFileName)
+    {
+        const std::string suffix = ".appx";
+        return (suffix.size() < inputFileName.size() &&
+                std::equal(suffix.rbegin(), suffix.rend(), inputFileName.rbegin()));
+    }
+
+	// For each of the appx files that we store in appxbundle, there is a corresponding
+	//	 entry in AppxBundleManifest.xml. This entry (an XML node) contains the "Offset"
+	//	 property which specifies the header offset in the final appxbundle file. Since this
+	//	 value is not known before we create the appxbundle, we provide a placeholder
+	//	 that looks like "FileName.appx-offset". This placeholder suggests that when packaging
+	//	 and creating the appxbundle we need to replace "FileName.appx-offset" with the
+	//	 number that represents the offset for FileName.appx.
+    inline
+    std::string
+	_ManifestContentsAfterPopulatingOffsets(const std::string &manifestInputFileName,
+                                           const std::vector<ZIPFileEntry> &otherEntries)
+    {
+        std::ifstream manifestInput(manifestInputFileName);
```

Comment:
As we talked, not having this file of having it in a wrong format would imply having a bad appxbundle file. However, the assumption is that whoever is using this for now will have to know about these peculiar details until we make the tool generate manifest on it's own.